packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -136,6 +137,7 @@ export default class SaytDriverPlugin implements Plugin {
  */
 export interface AutocompleteRequestConfig extends QueryTimeAutocompleteConfig {
   query: string;
+  config: any;
```

Comment:
For the record, these properties should have doc comments attached to them, but you don't need to do them here because we have another ticket with proper comments already.